src/scss/01_base/_helpers.scss

```diff
@@ -45,6 +45,15 @@
   }
 }
 
+@function containerWidth($container) {
```

Comment:
Questa funzione non ha senso. Basterebbe fare `map-get($containerConfig, $container);`

Reply:
però risolve il caso 0 - xsmall in un punto univoco e riutilizzabile (cosa che però non sta facendo adesso). Propongo di rifattorizzarla `containerWidth(breakpoint:null)` se null restituisce calc(100% - #{$gutter} - #{$gutter}); altrimenti la width definita per il container